include/swift/Parse/Parser.h

```diff
@@ -957,7 +962,8 @@ class Parser {
 
   ParserResult<Decl> parseDecl(bool IsAtStartOfLineOrPreviousHadSemi,
                                bool IfConfigsAreDeclAttrs,
-                               llvm::function_ref<void(Decl *)> Handler);
+                               llvm::function_ref<void(Decl *)> Handler,
+                               bool fromASTGen = false);
```

Comment:
Could you add a comment what `fromASTGen` does? Similar for the other functions that have `fromASTGen`.

---

Also, all the other parameter start with uppercase.

Reply:
I _think_ we are slowly migrating to "camelCase" for function parameters. I prefer new parameters being lower camel case.

Reply:
Disclaimer: I don’t know what the transition path is here but at least I have always tried to be consistent with the rest of the surrounding code. But I’ll leave it up to you, ultimately.